pkg/util/cmdline.go

```diff
@@ -98,49 +97,56 @@ func toNetworkInterfaces(data map[string]interface{}) error {
 // - "name: ens3"
 // - "ab:cd:ef:gh:ij:kl"
 // - "ens3"
-// and returns a map of either
-// "hwAddr: ab:cd:ef:gh:ij:kl"
-// or
-// "name: ens3"
-func parseIfDetails(details string) (*map[string]interface{}, error) {
-	var (
-		parts = make([]string, 0, 7)
-		data  string
-	)
-
-	for _, s := range strings.Split(details, ":") {
-		parts = append(parts, strings.TrimSpace(s))
+// - "hwAddr:ab:cd:ef:gh:ij:kl,name:ens3"
+// - "hwAddr:ab:cd:ef:gh:ij:kl,ens3"
```

Comment:
It is not a big problem, but would be nice to change.

The MAC address is using format like `00:18:36:aa:ee:ff`, `0..9`, `a..f` are valid char, could we amend the related codes and examples with valid MAC format? thanks.